### PR TITLE
More process tree comments. Fixed isUserEntered and getChildren(verboseMode) functions.

### PR DESCRIPTION
**Pull request by mitodrummer**
-lint fixes as well

### CODE REVIEW

**Comment by mitodrummer** [comment]
@animehart this has the new getChildren function that should fix the issue you are seeing.

**Review by Tacklebox — APPROVED** [review]
LGTM

File: x-pack/plugins/session_view/public/components/process_tree_node/index.tsx [hunk 1]
```diff
@@ -75,7 +75,7 @@ export function ProcessTreeNode({
   const { tty } = processDetails.process;
 
   const renderChildren = () => {
-    const children = process.getChildren(showGroupLeadersOnly);
+    const children = process.getChildren(!showGroupLeadersOnly);
```
Comment (mitodrummer) [inline x-pack/plugins/session_view/public/components/process_tree_node/index.tsx:78]: this call will be updated to take a verboseMode:boolean property. @animehart is working on this change as part of the verbose mode and timestamp toggle work.

**Review by opauloh — APPROVED** [review]
lgtm!!

**Review by zizhouW — APPROVED** [review]

**Pull request merged**